src/mono/mono.proj

```diff
@@ -50,6 +50,13 @@
     <MonoCxxCompiler Condition="'$(MonoCCompiler)' == 'gcc'">g++</MonoCxxCompiler>
   </PropertyGroup>
 
+  <!-- default metadata update support for specific platforms -->
+  <PropertyGroup>
+    <MonoMetadataUpdate Condition="'$(MonoMetadataUpdate)' == '' and ('$(TargetsAndroid)' == 'true' or '$(TargetsBrowser)' == 'true')">true</MonoMetadataUpdate>
+
+    <MonoMetadataUpdate Condition="'$(MonoMetadataUpdate)' == ''">false</MonoMetadataUpdate>
+  </PropertyGroup>
```

Comment:
nit: if you put this in `src/mono/Directory.Build.props` then you don't need to duplicate it in System.Private.CoreLib.csproj